### PR TITLE
Fix Play button style issues

### LINKED ISSUE #7397: Videos: Playback button placement and sizing issues
## Bug Description

When a video asset is resized to a small size, the playback button is displayed outside of the asset, directly above it. Currently, there's a couple of issues with this presentation:

1. Button is presented with scrollbars - 
Screen Shot 2021-05-05 at 3 03 41 PM

2. RTL changes the centering of the button and the way it's presented and the button is cutoff.
Screen Shot 2021-05-05 at 3 04 15 PM

3. Even for a large video, the button is still displayed off-center in RTL. Note that this will still happen, albeit slightly differently, after #7376 is merged.
Screen Shot 2021-05-05 at 10 39 18

## Additional Context

- Plugin Version:
- WordPress Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by miina**
## User-facing changes
| Before | After |
|-------|-------|
| Screenshot 2021-05-21 at 13 08 39 | Screenshot 2021-05-21 at 13 05 35 |
| Screenshot 2021-05-21 at 13 08 16 | Screenshot 2021-05-21 at 13 07 05 |

## Testing Instructions
I
1. Set your computer to always show scrollbars
2. Add a video element
3. Make the video element narrow enough for the play button to show above it instead of on top of it.
4. Verify the play button does not show scrollbars.

II
1. Switch the editor to RTL
2. Add a video element
3. Select the video element and hover to see the play button, verify it displays centred in the middle of the video.

### QA
See above.

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

### UAT

- [ ] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.
## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [ ] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #7397

### CODE REVIEW

File: assets/src/edit-story/elements/video/controls.js [hunk 1]
```diff
@@ -60,6 +61,7 @@ const ButtonWrapper = styled.div.attrs({ role: 'button', tabIndex: -1 })`
   pointer-events: initial;
   width: ${PLAY_BUTTON_SIZE}px;
   height: ${PLAY_BUTTON_SIZE}px;
+  overflow: hidden;
```
Comment (merapi) [inline assets/src/edit-story/elements/video/controls.js:64]: Right now it doesn't show scrollbars 👍🏻 , but the drop shadow is also cut, do we want to fix it too? (either enlarge the element and add padding for shadow or make changes in the parent?)
image

Comment (miina) [reply assets/src/edit-story/elements/video/controls.js:64]: Adjusted:
Screenshot 2021-05-25 at 14 05 33

Comment (merapi) [reply assets/src/edit-story/elements/video/controls.js:64]: @miina should we move it up a little? that should be easier/faster than z-index change.
image

Comment (miina) [reply assets/src/edit-story/elements/video/controls.js:64]: Hm, the shadow area shouldn't really cover the handle visibly though, I think. Will move it for correctness, thanks for the detailed check!

Comment (miina) [reply assets/src/edit-story/elements/video/controls.js:64]: Done!

Comment (merapi) [reply assets/src/edit-story/elements/video/controls.js:64]: > Hm, the shadow area shouldn't really cover the handle visibly though, I think. Will move it for correctness, thanks for the detailed check!

For me visually it was fine, but it was covering that little part of the handle - making it unactionable (and would trigger play/pause). All great now.

**Review by merapi — APPROVED** [review]

File: assets/src/edit-story/elements/video/controls.js [hunk 2]
```diff
@@ -255,7 +260,7 @@ function VideoControls({
             onMouseDown={handlePlayPause}
             isAbove={isPlayAbove}
           >
-            {isPlaying ? <Pause /> : <Play />}
+            <Icon isRTL={isRTL} />
```
Comment (swissspidy) [inline assets/src/edit-story/elements/video/controls.js:263]: Should this be a transient prop or isn't this a problem?

Comment (miina) [reply assets/src/edit-story/elements/video/controls.js:263]: `isRTL` is not a problem.

**Pull request merged**